docs/auth/atlassian/provider.md

```diff
@@ -47,8 +47,15 @@ auth:
         clientId: ${AUTH_ATLASSIAN_CLIENT_ID}
         clientSecret: ${AUTH_ATLASSIAN_CLIENT_SECRET}
         scope: ${AUTH_ATLASSIAN_SCOPES}
+        signIn:
+          resolvers:
```

Comment:
Some open questions that I would expect to find answer on this doc page:
- Are these all resolvers available? If not where can I find a comprehensive list?
- Can I also create custom ones and use them in this provider?

Reply:
I've added a dedicated resolvers section to each of these to address these two items 👍